_config.yml

```diff
@@ -38,11 +38,11 @@ people:
   position: TEST
```

Comment:
@priyasilverstein Do you want to add that you are doing a post-doc at the moment in front of the position section?